docs/pages/guides/splash-screens.mdx

```diff
@@ -30,6 +30,8 @@ You can work off of [this Sketch template](https://github.com/expo/files/blob/b2
 
 Export the image as a PNG and put it in your project directory. I'll assume it's in the **assets** directory and named **splash.png**.
 
+**Note:** Currently only PNG images are supported. If you use another image format, making a production build of your app will fail.
```

Comment:
```suggestion
> **warning:** Currently only PNG images are supported. If you use another image format, making a production build of your app will fail.
```

This sounds like a warning rather than general info and I think adding the warning callout type will provide a better experience.